src/main/kotlin/mate/academy/figure/Figure.kt

```diff
@@ -0,0 +1,8 @@
+package mate.academy.figure
+
+import mate.academy.behavior.Drawing
+import mate.academy.model.Color
+
+abstract class Figure(val name: String, val color: Color) : Drawing {
```

Comment:
It is better to move 'name' parameter to child classes as it is not used directly.

As this parameter is used in draw() method 'name' parameter could be taken from specific figure